canonical-web-and-design/maas-ui
# Reconnect to an existing websocket in the legacy app

## Done
- Reconnect to an existing websocket in the legacy app.
- Share the websocket between the bootstrap and region modules.

## QA
- Load a legacy page with the network details open in the dev tools.
- Switch between the react and angular clients a few times.
- The app should not reopen a websocket for the legacy client and you should only end up with two websocket connections.

## Fixes
Fixes: https://github.com/canonical-web-and-design/maas-ui/issues/1203.
See: https://github.com/canonical-web-and-design/maas-ui/issues/1211
